scripts/libgomp/create.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+# Builds a debian docker image with libgomp1 installed and copies the binary to the output path.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+DIR=$(dirname "$0")
+OUTPUT_PATH=$DIR/../../zoltar-xgboost/src/main/resources/lib/libgomp.so.1
+IMAGE_NAME=debian-libgomp1
+
+docker build -t $IMAGE_NAME $DIR
+
+docker cp -L $(docker create $IMAGE_NAME):/lib/x86_64-linux-gnu/libgomp.so.1 $OUTPUT_PATH
```

Comment:
Do we need mac version?

Reply:
@ravwojdyla according to the docs `OpenMP` seems to be not enabled by default.

https://xgboost.readthedocs.io/en/latest/jvm/index.html#enabling-openmp-for-mac-os

Reply:
mentioned file https://github.com/dmlc/xgboost/blob/master/jvm-packages/create_jni.py#L73

Reply:
given this not sure if we should worry about this for now.

Reply:
@regadas dope! have you tested it?

Reply:
@ravwojdyla yup, tests also confirm it, `OpenMP` is loaded by default for `xgboost` on linux.

https://github.com/spotify/zoltar/blob/cbfc4be173270f731db9ae2029291a01d30a3aa7/zoltar-xgboost/src/main/java/com/spotify/zoltar/xgboost/XGBoostModel.java#L59-L67